test/RelayHubValidator.test.ts

```diff
@@ -0,0 +1,72 @@
+require('source-map-support').install({ errorFormatterForce: true })
+
+const TestRelayHubValidator = artifacts.require('TestRelayHubValidator')
+
+contract('RelayHubValidator', ([from, senderAddress, target, paymaster, relayWorker, forwarder]) => {
+  let validator: any
+  before(async () => {
+    validator = await TestRelayHubValidator.new()
+  })
+
+  it('#len1 should return dynamic structure size', async () => {
```

Comment:
please start with 'should' at all costs

Reply:
sorry, English no good for that. its a sanity check that the method does its work.
`#len1` should be enough, though